Description: Return exit code 0 if no dir is in config file
 Return the exit code 0 for "asd sync" if no directory for sync
 is set in the config file /etc/asd.conf.
Author: Jan Luca Naumann <j.naumann@fu-berlin.de>
Forwarded: not-needed
Last-Update: 2016-10-27
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/common/anything-sync-daemon.in
+++ b/common/anything-sync-daemon.in
@@ -146,7 +146,7 @@
 	# nothing to do if these are empty
 	if [[ -z "${WHATTOSYNC[0]}" ]]; then
 		echo -e " ${BLD}Must define at least one directory in ${NRM}${BLU}$ASDCONF"${NRM}
-		exit 1
+		exit 0
 	fi
 
 	# make sure the user defined real dirs
